src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    public List<Bulldozer> get() {
+        Bulldozer bull1 = new Bulldozer("type1", 1);
+        Bulldozer bull2 = new Bulldozer("type2", 2);
+        Bulldozer bull3 = new Bulldozer("type3", 3);
+        return List.of(bull1, bull2, bull3);
```

Comment:
```suggestion
        return List.of(new Bulldozer("type1", 1), new Bulldozer("type2", 2), new Bulldozer("type3", 3));
```
A matter of taste, up to you